pkgs/applications/misc/tuckr/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tuckr";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "RaphGL";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-mpI0iAGMIzGGdObH5bfyA3iioNdquzLDZoSWxbAOsJ0=";
+  };
+
+  cargoPatches = [ ./fix-build.patch ]; # Cargo.lock is not up-to-date
+
+  cargoSha256 = "sha256-tm8fS8IWxWF4Vh+3QaCiruglZijdOic34vfAyxflDNM=";
+
+  doCheck = false; # test result: FAILED. 5 passed; 3 failed;
+
+  meta = with lib; {
+    description = "A super powered replacement for GNU Stow";
+    homepage = "https://github.com/RaphGL/Tuckr";
+    license = licenses.gpl3;
+    platforms = platforms.all;
```

Comment:
```suggestion
```

Rust does not run on all platforms. So just ignore it (except when the upstream is more restrictive, like "it runs only in Linux").